its/src/test/java/org/sonarsource/scanner/ant/it/AntTest.java

```diff
@@ -229,7 +228,7 @@ private static Component getComponent(String componentKey) {
   private static String getProjectVersion(String componentKey) {
     // Waiting for SONAR-7745 to have version in api/components/show, we use internal api/navigation/component WS to get the component
```

Comment:
looks like https://jira.sonarsource.com/browse/SONAR-7745 is done, if you got time you could use the new api. I know this is not in the scope of the orchestrator update, but since we are cleaning deprecated code... I guess stop using an internal API is nice to have